packages/ng/src/app/user/picture/user-picture.component.ts

```diff
@@ -11,15 +11,24 @@ import { LuUserDisplayPipe } from '../display';
 	styleUrls: ['./user-picture.component.scss'],
 })
 export class LuUserPictureComponent {
-
+	/**
+		 * User Display format.
+		 * It is set to 'fl' by default
+		 */
+	private _displayFormat = 'FL';
+	@Input()
+	set displayFormat(displayFormat: string) {
+		this._displayFormat = displayFormat.toUpperCase();

```

Comment:
why use this if you only want to support formats with initials only. If you tell me i can use the same formats from displyUser pipe, then i can use all of them

Reply:
If i want to display only the user firstname, i cant

Reply:
![image](https://user-images.githubusercontent.com/10089239/37395284-2c603fac-2776-11e8-87bc-b7e0c06ac42d.png)

Reply:
It's a style decision, we want this to display initials only

Reply:
yeah but its kind of a _hack_ that `toUpperCase` gives only the initials